src/command_modules/azure-cli-interactive/HISTORY.rst

```diff
@@ -3,9 +3,11 @@
 Release History
 ===============
 
-0.3.2 (unreleased)
+0.3.3 (unreleased)
 ++++++++++++++++++
+
 * --progress flag
+* Change the '?' gesture to inject into the next command
```

Comment:
Does it change the previous behavior or enhance it?  It used to be a I could do something like `?.id` to print just the id from the previous output.  Does that not still work?